frontend/src/css/modules/challenge.scss

```diff
@@ -284,3 +284,11 @@ md-select .md-select-value span:first-child:after {
 .margin-bottom-cancel {
     margin-bottom: 0px;
 }
+
+.watermarked {
+    background-image: linear-gradient(rgba(255,255,255,0.9), rgba(255,255,255,0.9)),url(../images/evalai-logo-centered.png);
```

Comment:
@matthew-so are you sure the logo path will work with the deployment as well? I think the path needs to be `dist/images/evalai-logo.png`. Can you please check other image URL usages and fix this?

Reply:
I am sure; relative URLs are regularly used in the frontend code, as can be seen here: ![relative_path_example_evalai](https://user-images.githubusercontent.com/42504035/187266379-3542f259-7ec2-4bea-bd30-d3a7e462d5c5.png)